drools-benchmarks-parent/drools-benchmarks-reliability/src/main/java/org/drools/benchmarks/reliability/AbstractReliabilityBenchmark.java

```diff
@@ -62,9 +66,19 @@ public String getInfinispanStorageMode() {
         }
     }
 
+    public enum Module {
+        INFINISPAN,
+        H2MVSTORE
+    }
```

Comment:
At first, I thought that `Module` is a good idea. But then we have to deal with unexpected combinations like "H2MVSTORE" + "REMOTE", or "H2MVSTORE" + "REMOTEPROTO".

If you run
```
java -jar target/drools-benchmarks-reliability.jar -jvmArgs "-Xms4g -Xmx4g" -foe true -wi 0 -i 1 -f 1 org.drools.benchmarks.reliability.InsertAndFireBenchmark
```
You will hit a failure with such a combination.

Probably it's better to use `Mode` only. So add "H2MVSTORE" to Mode. Maybe also it's better to rename like "EMBEDDED" to "INFINISPAN_EMBEDDED" and so on (but `infinispanStorageMode` values shouldn't be changed , because it's used for INFINISPAN_STORAGE_MODE)

Reply:
yes, I understand. I will make the changes needed.